### PR TITLE
44381 sql cursor

### PR DESCRIPTION
**Pull request by rhyshort**
When getting a document revision from a cursor, the column name
should be used rather than the expected index.

BugzId: 44381

reviewer @tomblench 
reviewer @emlaver

### CODE REVIEW

File: sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java [hunk 1]
```diff
@@ -1678,20 +1678,21 @@ private void setCurrent(SQLDatabase db,BasicDocumentRevision winner, boolean cur
...
     private static BasicDocumentRevision getFullRevisionFromCurrentCursor(Cursor cursor,
                                                                           List<? extends Attachment> attachments) {
-        String docId = cursor.getString(0);
-        long internalId = cursor.getLong(1);
-        String revId = cursor.getString(2);
-        long sequence = cursor.getLong(3);
-        byte[] json = cursor.getBlob(4);
-        boolean current = cursor.getInt(5) > 0;
-        boolean deleted = cursor.getInt(6) > 0;
+        //docs.docid, docs.doc_id, revid, sequence, json, current, deleted, parent
```
Comment (tomblench) [inline sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java]: Is this comment needed? we can see the column names on each line now

Comment (rhyshort) [inline sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java]: oops, no it was for me to work out the names of the columns

**Comment by tomblench** [comment]
OK +1

**Comment by emlaver** [comment]
+1

**Pull request merged**